packages/simplebar-react/index.js

```diff
@@ -10,7 +10,9 @@ export default function SimpleBar({ children, scrollableNodeProps, ...options })
         </div>
         <div className="simplebar-mask">
           <div className="simplebar-offset">
-            <div {...scrollableNodeProps} className={`simplebar-content${scrollableNodeProps && scrollableNodeProps.className ? ` ${scrollableNodeProps.className}` : ''}`}>{children}</div>
+            <div className="simplebar-resize-wrapper">
+              <div {...scrollableNodeProps} className={`simplebar-content${scrollableNodeProps && scrollableNodeProps.className ? ` ${scrollableNodeProps.className}` : ''}`}>{children}</div>
+            </div>
```

Comment:
Here the `resize-content` is inside `resize-wrapper` and it should be the opposite! Should be like:
```
<div className="simplebar-content">
      <div className="simplebar-resize-wrapper" {...scrollableNodeProps}>
            {children}
      </div>
</div>
```